### PR TITLE
Readme updates

### PR DESCRIPTION
**Pull request by elboulangero**

### CODE REVIEW

**Comment by tclavier** [comment]
@tessus I thinks, you can merge this PR.

File: README.md [hunk 1]
```diff
@@ -70,6 +70,10 @@ to the `dependencies` in your `component.json`.
   - **macOS:** no extra step required
 - [Bundler](http://bundler.io/) (Run `gem install bundler` to install).
 - [Node Package Manager (aka. `npm`)](https://docs.npmjs.com/getting-started/installing-node)
+- Tools required to build the font:
+  - **Ubuntu:** `sudo apt-get install fontforge woff-tools woff2`
+  - **Windows:** TODO
+  - **macOS:** TODO
```
Comment (tessus) [inline README.md:76]: Either add proper text (instead of TODO) or remove those 2 lines please.

File: README.md [hunk 2]
```diff
@@ -78,7 +82,7 @@ From the root of the repository, install the tools used to develop.
 
 Build the font:
 
-    $:/src/icons make
+    $ make -C src/icons
```
Comment (tessus) [inline README.md:85]: Technically the make command was not wrong.

`$:/src/icons` denotes the directory. I agree, it looks a bit strange and probably should rather look like `[src/icons]$ make`.
But yes, maybe your command is cleaner.

File: README.md (hunk 1 above)
Comment (elboulangero) [reply README.md]: Just removed these two lines.

File: README.md (hunk 2 above)
Comment (elboulangero) [reply README.md:83]: I have never seen this syntax before, the `:` at the beginning. So I even tried to type it in the terminal to see if it worked :)

In any case, `make -C` is a common way to invoke make in a subdir. `cd src/icons && make && cd -` also works, but it's less concise and elegant.

**Comment by tessus** [comment]
> I have never seen this syntax before,

It's not a syntax, but a prompt. Or at least I think that's what the original author intended it to be. But I get your point.

File: README.md (hunk 1 above)
Comment (tessus) [reply README.md]: thanks.

File: README.md (hunk 2 above)
Comment (tessus) [reply README.md:83]: Right, I know that. I'm just saying that the prompt (even if it looks weird) is not a wrong statement.
But ok, your make -C looks cleaner.

**Pull request merged**